src/libs/java/gradle.properties

```diff
@@ -1 +1 @@
-syslVersion=0.1.4-alpha-5
+syslVersion=0.1.4
```

Comment:
Unfortunately I issued this PR before the one #68 with the script update to automate this was merged.
Hopefully this won't be necessary again in the future.